test/e2e/run_passwd_test.go

```diff
@@ -125,4 +125,16 @@ USER 1000`, ALPINE)
 		Expect(session).Should(Exit(0))
 		Expect(session.OutputToString()).To(Not(ContainSubstring("/etc/group")))
 	})
+
+	It("podman run --no-manage-passwd flag", func() {
+		run := podmanTest.Podman([]string{"run", "--user", "1234:1234", ALPINE, "cat", "/etc/passwd"})
```

Comment:
You could simplify this by just doing
grep 1234:1234 /etc/passwd and expecting no failure.
But this looks good.

Reply:
wouldn't `grep 1234:1234 /etc/passwd` work even if 1234:1234 isn't there? It just wouldn't output anything.

Reply:
```
$ grep 1234:1234 /etc/passwd 
$ echo $?
1
```

Reply:
I am reverting to the cat version. The testing suite is consistently failing with the grep version but not on the grep itself, the entire command explodes. Very strange since the non `--passwd` one with grep works. 

Tested the command myself and grep outputs zero.